hive/configs/minatar/rainbow_circular_buffer.yml

```diff
@@ -15,9 +15,9 @@ save_dir: 'experiment'
 
 # Environment config
 environment:
-  name: 'MinAtarEnv'
+  name: 'GymEnv'
   kwargs:
-    env_name: 'breakout'
+    env_name: 'Breakout-v0'
```

Comment:
I think this should be `MinAtar/Breakout-v0`. Am I right?

Based on the instructions given [here](https://github.com/kenjyoung/MinAtar#openai-gym-wrapper).